src/app.js

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios';
+
+// kritieke vragen
+// 1. valuta check werkt niet bij minder dan 2? Teller werkt niet bij 1 valuta
+// 2. pagina verversen/resultaat opschonen 2de keer zoeken?
+
+
+let fieldValue = '';
+
+// html locaties
+const imageWrapper = document.getElementById('image-wrapper')
+const nameAndFlag = document.getElementById('name-and-flag')
+const searchContainer = document.getElementById('search-container')
+const inputForm = document.getElementById("user-search-bar")
+
+// event listeners
+inputForm.addEventListener('keyup', dataField)
```

Comment:
Je hebt in principe niets aan de waarde totdat de submit is afgegaan. Ik zou daarom in de handleSubmit functie de waarde pas uitvragen van het html element